twim/calibration/R3BTwimMapped2Cal.cxx

```diff
@@ -191,97 +236,86 @@ void R3BTwimMapped2Cal::Exec(Option_t*)
         return;
 
     auto** mappedData = new R3BTwimMappedData*[nHits];
-    Int_t secId = 0;
-    Int_t anodeId = 0;
-    Double_t pedestal = 0.;
-    Double_t slope = 1.;
 
-    for (Int_t s = 0; s < fNumSec; s++)
-        for (Int_t i = 0; i < (fNumAnodes + fNumAnodesRef + fNumAnodesTrig); i++)
+    for (Int_t sec = 0; sec < fNumSec; sec++)
+    {
+        fTwimCal[sec]->Init();
+    }
+    for (Int_t i = 0; i < nHits; i++)
+    {
+        mappedData[i] = dynamic_cast<R3BTwimMappedData*>(fTwimMappedDataCA->At(i));
```

Comment:
I'm confused about why we need a list of pointers to the mappedData. If I replace it with
```cpp
auto* mappedData = dynamic_cast<R3BTwimMappedData*>(fTwimMappedDataCA->At(i));
```
I don't see any problem with this.